src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
```

Comment:
We tend to not specify the values when they are sequential especially when the native code header already use this facility: 

```c
typedef enum _SECURITY_IMPERSONATION_LEVEL {
    SecurityAnonymous,
    SecurityIdentification,
    SecurityImpersonation,
    SecurityDelegation
    } SECURITY_IMPERSONATION_LEVEL, * PSECURITY_IMPERSONATION_LEVEL;
```

As far as re-using `SECURITY_IMPERSONATION_LEVEL` it's what the original header file does so it's more readable to port as close as possible but not essential. I was just pointing it out as one of the discutable point of the PR (Should I have used the computed value of 0 instead of `0 << 16` here for example.